graylog2-web-interface/src/components/widgets/Widget.jsx

```diff
@@ -204,37 +207,34 @@ const Widget = React.createClass({
       return <span />;
     }
     const showConfigModal = (
-      <WidgetConfigModal ref="configModal"
-                         dashboardId={this.props.dashboardId}
+      <WidgetConfigModal dashboardId={this.props.dashboardId}
```

Comment:
We still use `configModal` ref to open the info modal dialog, so it can't be removed.